nix/lexical.nix

```diff
@@ -0,0 +1,34 @@
+{ mixRelease
+, fetchMixDeps
+, erlang
+}: mixRelease rec {
+  pname = "lexical";
+  version = "development";
+
+  src = ./..;
```

Comment:
What about using [`pkgs.lib.cleanSource`](https://github.com/NixOS/nixpkgs/blob/master/lib/sources.nix#L55) on these `src` references? It cleans unnecessary stuff from the resulting derivation, such as git repositories.

There's also a way to ignore items on the gitignore on the resulting derivation using [`pkgs.gitignore`](https://nixos.org/manual/nixpkgs/stable/#sec-pkgs-nix-gitignore-usage)